src/Functions/getIntensityPeaks.m

```diff
@@ -1,12 +1,34 @@
-%% Get peaks respectivly center points at threshold below the peaks from 1D
-% intensity profile. Expected peakWaveCenter number == 4 for now (needs update to support electrodes with more ring contacts)!
-%
-% Andreas Husch
-% Centre Hospitalier de Luxembourg, Dep. of Neurosurgery /
-% University of Luxembourg - Luxembourg Centre for Systems Biomedicne
-% 2014 - 2017
-% mail@andreashusch.de, husch.andreas@chl.lu
 function [peakLocs, peakWaveCenters, peakValues, threshIntensityProfile, threshold, contactAreaCenter, contactAreaWidth, xrayMarkerAreaCenter, xrayMarkerAreaWidth] = getIntensityPeaks(intensityProfile, skelScaleMm, filterIdxs)
+% Get peaks, respectively center points, at threshold below the peaks from 1D
```

Comment:
@adhusch I always take your file header to put in the function description as it suits it quite well, but in this case this header appears to be more akin to a comment than a description. Maybe you could provide a more concise description, or tell me what i should edit out/edit in?

Reply:
Well, that function is a clean code disaster anyway, has way to many return values ;) Should be refactored in total..

Reply:
peakLocs 1d peak locations, 
peakWaveCenters 1d peak wave center locations
peakValues peak values 
threshIntensityProfile thresholded intensity profile 
threshold theshold
contactAreaCenter center of the assumed contact area
contactAreaWidth with of the assumed contact area
 xrayMarkerAreaCenter center of the xray marker (if any)
 xrayMarkerAreaWidth width of the xray maker (if any)